tests/db_integrity/test_data_updates.py

```diff
@@ -6,108 +6,80 @@
 from openghg.store.base import Datasource
 from openghg.retrieve import get_flux
 from openghg.retrieve import search
-from openghg.standardise import standardise_footprint
+from openghg.standardise import standardise_footprint, standardise_flux, standardise_surface
 from openghg.objectstore import get_bucket
-
-
 from helpers import clear_test_stores
 
 
-def flux_data_read():
-    """
-    Flux data set up.
+@pytest.fixture(autouse=True)
+def clear_stores():
+    """Clear the test store at the start of each test.
+
+    This will run before any other fixtures in the same scope (i.e. function scope).
+    In particular, it will run before all of the set-up tests below.
     """
-    # Emissions data
-    # Anthropogenic ch4 (methane) data from 2012 for EUROPE
-    source1 = "anthro"
-    domain = "EUROPE"
+    clear_test_stores()
 
-    emissions_datapath1 = get_emissions_datapath("ch4-anthro_EUROPE_2012.nc")
 
+@pytest.fixture(autouse=True, scope="module")
+def auto_bucket():
+    """Put `bucket` in namespace.
+
+    Making bucket a module level variable doesn't work, and
+    autouse fixtures don't provide return values unless passed
+    as arguments to the test, so this seems like the best way
+    to do this.
+    """
+    global bucket
```

Comment:
Can I ask what problem this is aiming to solve? Is the idea to allow `bucket` to be accessible to all tests without having to specify it each time?

I think overall we should avoid global state where we can as it can easily cause confusion. Using a fixture with the functions directly and without the autouse may be a more clear approach here.

Reply:
It's the same as calling "bucket" as a fixture each time. I could make it function scope and then it would essentially be a local variable.

I guess it serves the same purpose as the fixture that clears test stores each time.

I could just call the fixture explicitly if you prefer that.

Reply:
Yes I think if we can avoid global state where we can, that would be better.